infra/tf/modules/node/main.tf

```diff
@@ -168,6 +175,21 @@ sudo systemctl restart google-cloud-ops-agent
 apt update -y
 apt install -y pigz
 
+# Add Docker's official GPG key:
+apt install -y ca-certificates curl
+install -m 0755 -d /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/debian/gpg -o /etc/apt/keyrings/docker.asc
```

Comment:
You really want to put this in the repo, or anyone who compromises download.docker.com can compromise your image, which rather obviates the point of checking signatures.

Reply:
Hey Richard, not really. I just did a proof of concept to see if the containerized version works. This has to be refactored properly Monday when I'm back. I just make this as draft pr to record what I'm doing